scripts/deploy_oak_functions_loader

```diff
@@ -20,17 +20,22 @@ docker build \
 
 docker push "${FUNCTIONS_DOCKER_IMAGE_NAME}:latest"
 
-# Build and push an individual example application built on Oak Functions.
+# Build and push the `hello_world` example application built on Oak Functions.
 
-readonly FUNCTIONS_EXAMPLE_DOCKER_IMAGE_NAME='gcr.io/oak-ci/oak-functions-example'
+cargo -Zunstable-options build --release \
```

Comment:
There is a `TODO(#1943)` comment on line 10 of this file. Can that be removed now?
(Actually, I think I should have removed it when I merged #1943)

Reply:
Yes I'll do another PR to replace commands with the corresponding runner versions.